posts/2023-01-05-n5.md

```diff
@@ -0,0 +1,61 @@
+---
+title: N5 Plugins for Fiji
+summary: N5 plugins provide chunked storage support for Fiji.
+thumbnail_url: https://raw.githubusercontent.com/janelia-cosem/openorganelle-blog/main/assets/n5-thumbnail.png
+carousel_url: https://raw.githubusercontent.com/janelia-cosem/openorganelle-blog/main/assets/n5_carousel.png
+tags: ["hdf5","n5","zarr","fiji"]
+authors: ["John Bogovic"]
+date: "2023-01-05T01:01"
+published: False
+---
+
+Data hosted on OpenOrganelle are accessible from [Fiji](https://imagej.net/software/fiji/)<sup>1</sup> using the [N5
+plugins](https://github.com/saalfeldlab/n5-ij)<sup>2</sup> and [N5 Viewer](https://github.com/saalfeldlab/n5-viewer) from the
+Saalfeld lab. N5 Viewer relies critically on [BigDataViewer](https://imagej.net/plugins/bdv/)<sup>3</sup>.
+
+The video below video shows how to: 
+1. explore the list of images for a given OpenOrganelle dataset
+2. select and view image data using N5-Viewer
+3. simple navigation using BigDataViewer [details here](https://imagej.net/plugins/bdv/#basic-navigation).
+3. crop a subset of the data and export it to Fiji / ImageJ.
+
+https://www.youtube.com/watch?v=F88pd-Q8D7Y
+
+In addition to its with OpenOrganelle, the N5 plugins for Fiji make it easy to read, write, and visualize many chunked data
+formats (HDF5<sub>4</sub>, N5, Zarr<sub>5</sub>) for both local and cloud storage. This video gives a short tutorial for basic
+reading and writing images from Fiji,
+
+https://www.youtube.com/watch?v=jwdKmDfDXXw
+
+More detailed descriptions of the options for image export are given here:
+
+https://www.youtube.com/watch?v=esxnmUQO2Q4
+
+For those interested in more details in a longer-form, including discussion of metadata, John Bogovic describes many more
```

Comment:
You are credited as the author, so maybe replace "John Bogovic" with "I (John Bogovic)" or something that makes it clear that you are John Bogovic :)